lib/browser/api/web-contents.js

```diff
@@ -280,6 +280,22 @@ WebContents.prototype.loadFile = function (filePath, options = {}) {
   }))
 }
 
+WebContents.prototype.enableEditHotkeys = function () {
+  if (process.platform === 'darwin') {
```

Comment:
Any reason we can't make this os agnostic?  Or does the issue not exist in OSes other than Mac?